pkg/config/app/kuma-cp/config.go

```diff
@@ -270,6 +270,10 @@ type GeneralConfig struct {
 	AdvertisedHostname string `yaml:"advertisedHostname" envconfig:"kuma_general_advertised_hostname"`
 	// DNSCacheTTL represents duration for how long Kuma CP will cache result of resolving dataplane's domain name
 	DNSCacheTTL time.Duration `yaml:"dnsCacheTTL" envconfig:"kuma_general_dns_cache_ttl"`
+	// TlsCertFile defines a path to a file with PEM-encoded TLS cert that will be used across all the Kuma Servers.
+	TlsCertFile string `yaml:"tlsCertFile" envconfig:"kuma_general_tls_cert_file"`
```

Comment:
Why not `TLSCertFile`? To be consistent with `DNSCacheTTL`

Reply:
It's consistent with other `TlsCertFile` properties of the config. I'd like to keep it this way for now